### ISSUE TITLE
IDEA 2020.3 Invalid runtime JDK 1.8

### ISSUE THREAD

**Issue by zd-dalibor** [denis-fokin/ChooseRuntime#19]
If idea cannot start anymore after runtime change how to rollback the change.

For example whit idea 2020.3 when I change runtime to 1.8 I got the error message that idea cannot start with 1.8 SDK.
How can I manually change the runtime JDK to the default one?

**Comment by filiagees** [comment]
@zd-dalibor , unfortunately, I also had this problem :(
![image](https://user-images.githubusercontent.com/4993994/111087168-d1b82d00-84fe-11eb-871c-8dc6ce38ea6f.png)


Lucky, I realized that the configuration related to JDK was stored in `~/.config/JetBrains/IdeaIC2020.3/idea.jdk`.
Then, after delete it, my intellij back to normal again :pray: 


Due to this issue, I disabled the plugin. 
And again, I realized that `com.jetbrains.ChooseRuntime` was appended to `~/.config/JetBrains/IdeaIC2020.3/disabled_plugins.txt` file.
So, it might also be an option to rollback (not tested).

I hope to help you!
:)
